spec/features/show_page_spec.rb

```diff
@@ -1,21 +1,20 @@
 require "rails_helper"
 
 RSpec.describe "customer show page" do
-
   it "displays the customers orders paginated" do
     customer = create(:customer)
-    first_order, _a, _b, last_order = Array.new(4) { create(:order, customer: customer) }
+    orders = create_list(:order, 4, customer: customer)
 
     visit admin_customer_path(customer)
     within('.attribute-data--has-many') do
-      expect(page).to have_content(first_order.id)
-      expect(page).to have_no_content(last_order.id)
+      expect(page).to have_order_row(orders.first.id)
+      expect(page).not_to have_order_row(orders.last.id)
     end
-
     click_on("Next ›")
+
     within('.attribute-data--has-many') do
-      expect(page).to have_content(last_order.id)
-      expect(page).to have_no_content(first_order.id)
+      expect(page).to have_order_row(orders.last.id)
+      expect(page).not_to have_order_row(orders.first.id)
     end
   end
```

Comment:
@jcmorrow What do you think about adding a context around those pagination specs to give a little further ... well, context ... as to why you would see some on one page versus the next?  Right now it's not clear to me when I first come to this index page that there's a limit per page, how many that limit is, how many I should be seeing, etc.